metadata.json

```diff
@@ -33,14 +33,13 @@
   "requirements": [
     {
       "name": "puppet",
-      "version_requirement": ">= 5.5.8 < 7.0.0"
+      "version_requirement": ">= 6.1.0 < 8.0.0"
```

Comment:
Not a problem, but we shouldn't forget to make issues for this so it's

Reply:
will do

Reply:
#616

Reply:
don't know what modulesync exactly does, but why does it require a higher puppet version?

Reply:
it does not perse,

 but it was easier to get this in while fixing the test matrix. Than to fix the matrix and than immediately after remove the support for things like puppet 5 and centos 6/